### PR TITLE
fix(Script) Quest "Deactivating the Spire"

### PR DESCRIPTION
**Pull request by malcrom**
## Changes Proposed:
- Update quest to deactivate the beams for 10 seconds on click like official.  
- I chose C++ over SAi for this scripting due to DespawnOrUnsummon forced respawn timer fails the first time in SAI and I wanted this to work correctly.

## SOURCE:
Sniff

## Tests Performed:
- Tested in game

## How to Test the Changes:
1. .quest add 8889
2. .go xyz 9334.2 -7880.1899 74.9103
3. do quest

### CODE REVIEW

File: src/server/scripts/World/go_scripts.cpp [hunk 1]
```diff
@@ -1928,6 +1928,30 @@ class go_bells : public GameObjectScript
...
+{
+    NPC_POWER_SOURCE_INVISIBLE_BUNNY = 17984
+};
+
+class go_duskwither_spire_power_source : public GameObjectScript
+{
+public:
+    go_duskwither_spire_power_source() : GameObjectScript("go_duskwither_spire_power_source") {}
+
+    bool OnGossipHello(Player* player, GameObject* go) override
```
Comment (acidmanifesto) [inline src/server/scripts/World/go_scripts.cpp:1945]:
```
bool OnGossipHello(Player* /*player*/, GameObject* go) override
```

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

File: src/server/scripts/World/go_scripts.cpp (hunk 1 above)
Comment (acidmanifesto) [reply src/server/scripts/World/go_scripts.cpp:1945]: commit out the **player** and that should fix the compile testing.

Comment (malcrom) [reply src/server/scripts/World/go_scripts.cpp]: Thanks, fixed

Comment (acidmanifesto) [reply src/server/scripts/World/go_scripts.cpp]: You forgot to delete player so it shows as Player* /*player*/ player.
Needs to show as Player* /*player*/

**Comment by acidmanifesto** [comment]
```
You forgot to delete player so it shows as Player* /*player*/ player.
Needs to show as Player* /*player*/
```

**Review by acidmanifesto — APPROVED** [review]

**Pull request merged**